DigitalLearningSolutions.Web/Controllers/ApplicationSelectorController.cs

```diff
@@ -13,7 +13,7 @@ public class ApplicationSelectorController : Controller
         public IActionResult Index()
         {
             var learningPortalAccess = User.GetCustomClaimAsBool(CustomClaimTypes.LearnUserAuthenticated) ?? false;
-            var trackingSystemAccess = User.GetCustomClaimAsBool(CustomClaimTypes.UserCentreAdmin) ?? false;
+            var trackingSystemAccess = User.GetCustomClaimAsBool(CustomClaimTypes.UserUserAdmin) | User.GetCustomClaimAsBool(CustomClaimTypes.UserCentreManager) | User.GetCustomClaimAsBool(CustomClaimTypes.UserCentreAdmin) ?? false;

```

Comment:
This should be able to be replaced with User.HasCentreAdminPermissions(), I think the resulting bool is the same.

If it can't be replaced by that smaller helper method, you need to run the formatter on this to break up the long line. If you don't know how to do that drop me a message.